advisor-go/src/quixo/simulation/simulation.go

```diff
@@ -0,0 +1,55 @@
+package simulation
```

Comment:
`simulation` => `advice` ?

Reply:
I like simulation because it's understandable that the package will test a lot of possibility. Maybe I should split that in two package in the future